app/models/course/assessment/assessment_ability.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 module Course::Assessment::AssessmentAbility
+  include Course::Assessment::Answer::AnswerAbility
```

Comment:
Just leaving this here first, the inclusion here of the answer ability might be overwritten by subsequent abilities declaration based on how Ruby does module inclusions and cancancan ability definition. We should make sure this is tested properly.